Lpd 42839 Add no_NO language | Norwegian (Norway)

thiago-buarqque opened the pull request:
https://liferay.atlassian.net/browse/LPD-42921

Due to a typo we missed adding the no_NO language id (see [Jira comment](https://liferay.atlassian.net/browse/LPD-42839?focusedCommentId=2808492))

liferay-continuous-integration commented:
The following guidelines have been set by the owner of this repository: 
- &nbsp;&nbsp;&nbsp;&nbsp;"Thank you for sending this PR to liferay-platform-experience. If you haven't already,take this opportunity to review our [pull request guidelines](https://liferay.atlassian.net/wiki/spaces/PEDS/pages/2915598365). For additional help,please reach out via the [appropriate channels](https://liferay.atlassian.net/wiki/spaces/PEDS/overview#%3Aslack%3A-Slack-Channels)."

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

algobob commented on modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:
```diff
@@ -10083,6 +10083,7 @@ language.mww=Hmong Daw
 language.nb=Norwegian
 language.nl=Dutch
 language.nl_BE=Dutch (Belgium)
+language.no_NO=Norwegian (Norway)
```
Question: Why no only `language.no=...` ? There are no other language variation for this, right?

brianwulbern commented:
![image (1)](https://github.com/user-attachments/assets/886ab1dd-0bb5-4085-bec4-721ea7ba20e6)

These are the options on the Crowdin side. We already have Norwegian Bokmal in portal, which we could change to [nb_NO](https://simplelocalize.io/data/locale-code/nb-NO/), but the languages will display their full names on Crowdin, so it won't confuse them if we have `nb` and `no_NO` internally. IMHO, if we use `no_NO` it contrasts better with us already having a language from Norway (nb)

brianwulbern commented:
But in thinking about it, I'm not sure if Crowdin's `no` will map automatically to our `no_NO` files on Github. But as `no-NO` is the [official code](https://simplelocalize.io/data/locale-code/no-NO/), it should be ok.

thiago-buarqque commented:
@brianwulbern actually I think @algobob is right, it should be `no` in the [Language.properties](https://github.com/liferay-platform-experience/liferay-portal/pull/857/files/8b55c9e9b84fbfd795c3956d7e4ce9698816c64e#diff-ef119629946c9b5876f3ddaf7e739fa3316781f38535fa10141db62509a5d684) file.

For instance, [when I added Macedonian](https://github.com/liferay-platform-experience/liferay-portal/pull/857/files/8b55c9e9b84fbfd795c3956d7e4ce9698816c64e#diff-ef119629946c9b5876f3ddaf7e739fa3316781f38535fa10141db62509a5d684R10080), I added mk instead of mk_MK

thiago-buarqque commented:
And since just `no` already refers for Norwegian, it should work fine in Crowdin

brianwulbern commented:
Sent follow-up pull to https://github.com/liferay-platform-experience/liferay-portal/pull/859

thiago-buarqque commented:
Thanks @brianwulbern
